website/docs/r/codebuild_webhook.html.markdown

```diff
@@ -8,13 +8,17 @@ description: |-
 
 # aws_codebuild_webhook
 
-Provides a CodeBuild Webhook resource.
+Manages a CodeBuild webhook, which is an endpoint accepted by the CodeBuild service to trigger builds from source code repositories. Depending on the source type of the CodeBuild project, the CodeBuild service may also automatically create and delete the actual repository webhook as well.
 
 ## Example Usage
 
 ### GitHub
 
-~> **Note:** For GitHub source projects, the AWS account that Terraform uses to create this resource *must* have authorized CodeBuild to access GitHub's OAuth API in each applicable region. This is a manual step that must be done *before* creating webhooks with this resource. If OAuth is not configured, AWS will return an error similar to `ResourceNotFoundException: Could not find access token for server type github`. More information can be found in the [CodeBuild User Guide](https://docs.aws.amazon.com/codebuild/latest/userguide/sample-github-pull-request.html).
+When working with GitHub (github.com) source CodeBuild webhooks, the CodeBuild service will automatically create (on `aws_codebuild_webhook` resource creation) and delete (on `aws_codebuild_webhook` resource deletion) the GitHub repository webhook using its granted OAuth permissions. This behavior cannot be controlled by Terraform.
```

Comment:
`github.com` -> `https://github.com` (if we need the link?)

Reply:
I'll update -- thanks.